scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -292,16 +294,35 @@ task ScatterIntervals {
     Int? preemptible_attempts
 
     Int machine_mem_mb = select_first([mem_gb, 2]) * 1000
+    Int command_mem_mb = machine_mem_mb - 500
+
+    # If optional output_dir not specified, use "out";
+    String output_dir_ = select_first([output_dir, "out"])
 
     String base_filename = basename(interval_list, ".interval_list")
 
     command <<<
         set -e
-
-        grep @ ${interval_list} > header.txt
-        grep -v @ ${interval_list} > all_intervals.txt
-        split -l ${num_intervals_per_scatter} --numeric-suffixes all_intervals.txt ${base_filename}.scattered.
-        for i in ${base_filename}.scattered.*; do cat header.txt $i > $i.interval_list; done
+        mkdir ${output_dir_}
+        export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk4_jar_override}
+        
+        {
+            >&2 echo "Attempting to run IntervalListTools..."
```

Comment:
Any reason you're redirecting this to stderr? Should we do this for other tasks?

Reply:
Cromwell directs GATK logs to stderr as well...just wanted to get all the messages in the same place.

Reply:
Hm, my python logs go to the stdout.log file, at least in Firecloud

Reply:
Actually, I think I misspoke above---GATK logs go to stderr regardless.  Currently, gCNV python output goes to stdout (see #4629).  However, note that the messages above are just echoed from bash.